packages/web-components/fast-ssr/src/element-renderer/elemenent-renderer.spec.ts

```diff
@@ -109,15 +109,36 @@ test.describe("FASTElementRenderer", () => {
             `);
         });
 
-        test("should render an attribute with no value when a boolean attr evaluates true", () => {
+        test("should render an attribute when a boolean attr evaluates true/false", () => {
             const { templateRenderer } = fastSSR();
             const result = consolidate(templateRenderer.render(html`
```

Comment:
Instead of changing this test, can you just add new ones? We still want to validate the scenario this test covers.

Reply:
Sure, will update in the next revision